src/condor_utils/cron_job_params.cpp

```diff
@@ -232,7 +232,7 @@ CronJobParams::InitPeriod( const std::string &param_period )
 		return false;
 	} else {
 		char	modifier = 'S';
-		int		num = sscanf( param_period.c_str(), "%d%c",
+		int		num = sscanf( param_period.c_str(), "%ld%c",
```

Comment:
time_t is a long long on Windows. I think you need to store the value in a local long long variable, then assign into m_period after the sscanf().